templates/shelf_firebase_jwt/TEMPLATE.md

```diff
@@ -0,0 +1,108 @@
+---
+name: Firebase + JWT Auth + Shelf
+description: Build an authentication flow using Firebase + JWT with your Backend from a Flutter application.
+tags: ["firebase", "shelf", "authentication", "jwt"]
+username: Invertase
+---
+
+# Firebase + JWT Auth + Shelf
+
+## Overview
+
+The example shows how to use Firebase + JWT Authentication in Flutter app & Dart Backend.
+
+### Features
+
+- [Shelf](https://pub.dev/packages/shelf)
+- [Flutter](https://flutter.dev)
+- [Firebase](https://firebase.google.com/)
+- [JWT Authentication](https://jwt.io/introduction)
+
+### Getting Started
+
+#### Bootstrap
+
+Initialize your project using the `shelf_firebase_jwt` brick
+
+```shell
+$ mason make shelf_firebase_jwt
+```
+
+#### Setup Firebase Project
+
+You must create a Firebase account and generate Firebase Admin Service Account and download.

```

Comment:
needs more instructions on how to do this

Reply:
Yeah, I've expanded on this.